server/printing/printing_server.py

```diff
@@ -36,7 +37,8 @@ def SendRequestToPrinter(self, encoded_file, copies=1, options={}):
         command += "tmp.pdf"
         status = os.popen(command)
         os.remove("tmp.pdf")
-        return 'printed' if status else 'error'
+        SCEPrinter(self.DeterminePrinterForJob)
+        return 'printed' if status else 'error', printer
```

Comment:
What's "printer" referring to? If the statement above `SCEPrinter(self.DeterminePrinterForJob)` returns a printer, you need to change it to `printer = SCEPrinter(self.DeterminePrinterForJob)`